parsers/jscript.c

```diff
@@ -1653,15 +1653,45 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 
 		if (isKeyword (token, KEYWORD_async))
 			readToken (token);
-		else if (isType(token, TOKEN_KEYWORD) && isKeyword (token, KEYWORD_get))
+		else if (isType (token, TOKEN_KEYWORD) &&
+				 (isKeyword (token, KEYWORD_get) || isKeyword (token, KEYWORD_set)))
 		{
-			is_getter = true;
-			readToken (token);
-		}
-		else if (isType(token, TOKEN_KEYWORD) && isKeyword (token, KEYWORD_set))
-		{
-			is_setter = true;
-			readToken (token);
+			if (NextToken == NULL)
+			{
+				tokenInfo *savedToken = newToken ();
+				copyToken (savedToken, token, true);
+				readToken (token);
+				if (isType(token, TOKEN_OPEN_PAREN) && NextToken == NULL)
+				{
+					NextToken = newToken ();
+					copyToken (NextToken, token, false);	/* save token for next read */
+					copyToken (token, savedToken, true);	/* restore token to process */
+					token->type = TOKEN_IDENTIFIER;			/* process as identifier */
+				}
+				else if (isKeyword (savedToken, KEYWORD_get))
+				{
+					TRACE_PRINT("getter");
+					is_getter = true;
+				}
+				else
+				{
+					TRACE_PRINT("setter");
+					is_setter = true;
+				}
+				deleteToken (savedToken);
+			}
+			else if (isKeyword (token, KEYWORD_get))
+			{
+					TRACE_PRINT("getter");
+				is_getter = true;
+				readToken (token);
+			}
+			else
+			{
+					TRACE_PRINT("setter");
+				is_setter = true;
```

Comment:
The style of indentation looks strange. Maybe this depends on a tool you use or the web interface of github.
Anyway, could you look into this?

Reply:
Fixed by removing the unintentional debugging code